index.html

```diff
@@ -386,6 +386,30 @@
           a representation of the signing date.</dd>
         </dl>
       </section>
+
+      <section>
+        <h2>Revocation Model</h2>
+        <p>A <a>Verifiable Claim</a> may provide revocation information
+        by referring to a revocation list by adding the following
+        <a>property</a> to the <a>Verifiable Claims Model</a>:</p>
+        <dl>
+          <dt><var>revocationList</var></dt>
+          <dd>The format of the revocation list will vary depending on the
+          signature scheme used, but will almost always be a URL to a
+          resource that contains zero or more revocation entries.
```

Comment:
"the signature scheme used, but will almost always be a URL" => "the revocation scheme used. One simple example format is a URL that contains zero or more revocation entries, where the entry format is specified via the revocation scheme."

1. I don't think we should say "will almost ways" anything, we don't know.
2. The revocation scheme determines the format, not the signature scheme (directly). It may be that a signature scheme specifies a recommended revocation scheme, but it is most directly specified through the revocation scheme.